contracts/locker/BaseLocker.sol

```diff
@@ -59,6 +66,7 @@ contract BaseLocker is
         uint256 _maxTime
     ) internal {
         __ERC721_init(_name, _symbol);
+        __Ownable_init(msg.sender);
```

Comment:
If we can change this to an address passed via the function parameter instead of `msg.sender` it'd be great because the proxy update functions are not called directly but instead via a `ProxyAdmin` contract.

So if this upgrade were to execute, `msg.sender` will be the `ProxyAdmin` contract and not the orignal sender.

Reply:
Done https://github.com/zerolend/governance/pull/35/commits/3f0ba6505725cf56d6141116a5ea3937729d07e6